src/wallet/wallet2.h

```diff
@@ -1281,6 +1290,10 @@ namespace tools
     void hash_m_transfer(const transfer_details & transfer, crypto::hash &hash) const;
     uint64_t hash_m_transfers(int64_t transfer_height, crypto::hash &hash) const;
     void finish_rescan_bc_keep_key_images(uint64_t transfer_height, const crypto::hash &hash);
+    inline void set_export_format(const ExportFormat& export_format)
+      {
+        m_export_format = export_format;
+      }
```

Comment:
Can you move this to wherever other options are ? It makes things simpler :)

Reply:
They are really all over the place, but I'm picking something better :) .

Reply:
You added the getter at the right place already, the options are pairs of functions next to each other :)